selectors.yml

```diff
@@ -27,12 +33,24 @@ selectors:
   2358:
     selector: 5990477251245693094
     name: "ethereum-testnet-sepolia-kroma-1"
+  4002: 
+    selector: 4905564228793744293
+    name: "fantom-testnet-1"
```

Comment:
Hmm.. so far, the `-1` suffix is named `rollup_instance` in `Network naming proposal` doc, so I'm not sure it applies to testnets of other L1s, like this.
Is this a L1 testnet, or a rollup of other network?

Reply:
this is an L1 testnet 

https://docs.fantom.foundation/wallet/tutorials/set-up-metamask-testnet

Reply:
I just keep it incase they release another testnet

Reply:
If they release another testnet, it's expected that each has their own naming, in which case you'd go with `$l1_name-testnet-$testnet_name` instead

Reply:
fixed! fantom doesnot have an explicit name for their testnet